modules/ROOT/pages/sap/sap-s4-hana-connector.adoc

```diff
@@ -0,0 +1,466 @@
+= SAP S/4 HANA Cloud Connector
```

Comment:
The name of this connector seems to be incorrect. On the SAP website the software is called SAP S/4HANA. We should change this for all occurrences of the doc and product name.